From 6a3a5088430880f05f1e5d13c6b260c02e4f6034 Mon Sep 17 00:00:00 2001
From: Jun Li <jun.li@nxp.com>
Date: Thu, 18 Jan 2018 00:16:57 +0800
Subject: [PATCH 3204/5242] MLK-17399 ARM64: dts: imx8mq: enable usb3 link PM

commit  8eb602957dd816153d814c643b157982e1f3425f from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable USB3 hardware link power management, so the link can enter
U1 and U2 if there is no data transfer if the deivce can support
them.

Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index c506a20..a791d3d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -887,6 +887,7 @@
 			power-domains = <&usb_otg1_pd>;
 			snps,power-down-scale = <2>;
 			usb3-resume-missing-cas;
+			usb3-lpm-capable;
 			status = "disabled";
 		};
 	};
@@ -927,6 +928,7 @@
 			power-domains = <&usb_otg2_pd>;
 			snps,power-down-scale = <2>;
 			usb3-resume-missing-cas;
+			usb3-lpm-capable;
 			status = "disabled";
 		};
 	};
-- 
1.7.9.5

